Update Rule “book-developers-for-a-project/rule” (#5998)
SSWConsulting/SSW.Rules.Content: rules/book-developers-for-a-project/rule.md

This is generally the easiest way for developers to book themselves in as they don't need to leave Outlook and 90% of the steps will already be familiar to them.

1. Create a new Outlook appointment (in the Desktop or Web client) and give it a relevant subject
2. Invite required resources if there are other people working with you or if you're booking for someone else  

>   **Note:** If you're booking for someone else, just have them in the To box of the appointment. If you're booking yourself AND someone else, have both of you in the To box

Reviewer: Hey @samsmithssw 

1. Change from

> the To box

to

> the "To:" field

>    **Note:** If you're booking for someone else, just have them in the To box of the appointment. If you're booking yourself AND someone else, have both of you in the To box

Reviewer: Hey @samsmithssw 

1. Change from

> the To box

to

> the "To:" field

Author: Done!

To create a Tentative Booking:

1. Follow the same steps above to create the Appointment

> 2. Instead of choosing a client account for 'Set Regarding', choose the company **ssw.pencilledin**.

Reviewer: Hey @samsmithssw 

As per Adam's email: **SSW Rules should be generic**

1. Make this sentence:

> 2. Instead of choosing a client account (e.g. Northwind) for 'Set Regarding', choose a pre-defined "company" account for tentatives (e.g. SSW.PencilledIn).

Author: Hey @tiagov8 

I don't think that rule works in this case as it's specifically talking about how you use the pencilled.in function. So there is no need to be generic in this case.

> 2. Instead of choosing a client account for 'Set Regarding', choose the client you have created for the purpose of tentative bookings (E.g.**ssw.pencilledin**)

Reviewer 2: ```suggestion
2. Instead of choosing a client account for 'Set Regarding', create and use a pre-defined account for the purpose of tentative bookings (E.g.**SSW.PencilledIn**)
```
